app/javascript/dashboard/components/layout/AvailabilityStatus.vue

```diff
@@ -55,21 +55,32 @@ export default {
     return {
       isStatusMenuOpened: false,
       isUpdating: false,
+      statusKeys: ['online', 'busy', 'offline'],
     };
   },
 
   computed: {
     ...mapGetters({
       currentUser: 'getCurrentUser',
     }),
+    availabilityDisplayLabel() {
+      const availabilityIndex = this.statusKeys.findIndex(
+        key => key === this.currentUserAvailabilityStatus
+      );
+      return this.$t('PROFILE_SETTINGS.FORM.AVAILABILITY.STATUSES_LIST')[
+        availabilityIndex
```

Comment:
Should we rely array index position from translation files, since its compared against order in our constant variable?